jupyterlab/labbox_ephys_widgets_jp/src/exampleSorting.tsx

```diff
@@ -40,21 +40,6 @@ const sorting: Sorting = {
                 "spike_sign": -1
             }
         }
-    },
-    "sortingInfo": {
```

Comment:
sortingInfo is no longer a field in the spec of Sorting. More about that further down.